code/iaas/logic/src/main/java/io/cattle/platform/process/lb/LoadBalancerUpdateAll.java

```diff
@@ -0,0 +1,33 @@
+package io.cattle.platform.process.lb;
+
+import io.cattle.platform.core.constants.LoadBalancerConstants;
+import io.cattle.platform.engine.handler.HandlerResult;
+import io.cattle.platform.engine.process.ProcessInstance;
+import io.cattle.platform.engine.process.ProcessState;
+import io.cattle.platform.process.common.handler.AbstractObjectProcessHandler;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+
+@Named
+public class LoadBalancerUpdateAll extends AbstractObjectProcessHandler {
+
+    @Inject
+    LoadBalancerSetHosts setHosts;
```

Comment:
Having properties that begin with `set` seems like it could lead to confusion, since it clashes with POJO get/set naming convention.

Not a blocker, up to you if you want to change it. Just wanted to point it out.